### PR TITLE
add Gravity Alpha Mainnet

### PR DESCRIPTION
**Pull request by JunayedMh**

### CODE REVIEW

File: gravitymainnet/logo.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 707.04 511.67">
+  <path d="M704.61 53.97c-5.08-8.77-16.39-11.66-25.06-6.4L214.24 330.18c-11.86-22.15-18.58-47.46-18.58-74.34 0-87.27 70.83-158.02 158.2-158.02 37.52 0 71.96 13.08 99.08 34.87l-98.99 67.87L611.21 52.21c5.54-3.19 7.44-10.28 4.23-15.8-3.38-5.84-11.01-7.57-16.59-3.75l-62.99 43.19C489.44 29.01 425.06 0 353.86 0 212.4 0 97.72 114.54 97.72 255.84c0 45.49 11.9 88.2 32.74 125.22L0 460.29l133.95-73.24c44.73 74.64 126.47 124.62 219.92 124.62 141.46 0 256.14-114.54 256.14-255.84v-30.55c0-23.93 9.72-54.55 44.18-72.99 2.95-1.58 4.02-5.28 2.35-8.19-1.64-2.83-5.26-3.81-8.09-2.17L343 318.08c34.1-14.48 60.16-23.28 94.71-30.21 71.97-14.44 73.8 19.83 46.01 58.49-18.61 25.88-60.65 67.45-129.81 67.45-56.4 0-105.88-29.47-133.9-73.83L697.63 78.84c8.91-4.86 12.05-16.1 6.98-24.88Z" style="fill:#000;stroke-width:0"/>
+</svg>
```
Comment (paulbalaji) [inline gravitymainnet/logo.svg:3]: run through svg minifier https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" data-name="Layer 1" viewBox="0 0 707 512"><path d="M705 54c-5-9-17-12-25-6L214 330a158 158 0 0 1 239-197l-99 68L611 52a12 12 0 1 0-12-19l-63 43a255 255 0 0 0-406 305L0 460l134-73a256 256 0 0 0 476-131v-31c0-24 10-54 44-73a6 6 0 1 0-6-10L343 318c34-14 60-23 95-30 72-15 74 20 46 58a159 159 0 0 1-264-6L698 79c9-5 12-16 7-25Z"/></svg>
```

File: gravitymainnet/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    domainId: 1625
+    isTestnet: false
+    name: gravitymainnet
+    nativeToken:
+      decimals: 18
+      name: Gravity
+      symbol: G
+    protocol: ethereum
+    rpcUrls:
+      - http: https://rpc.gravity.xyz
```
Comment (paulbalaji) [inline gravitymainnet/metadata.yaml:17]: you should format this properly

**Review by paulbalaji — COMMENTED** [review]
appreciate the recent PRs! would ask that you also pay detail to review comments being made and keep them in mind for each submission. For example:
- running logos through an svg minifier like https://jakearchibald.github.io/svgomg/
- running `yarn lint:fix` and `yarn prettier` 
- ensuring there is a changeset by running `yarn changeset add`

**Comment by JunayedMh** [comment]
thanks,i did edit all 2 files but i dont know how to add or create changeset file..in my terminal the command not creating anything.

**Comment by JunayedMh** [comment]
And i did submitted the logo running through https://jakearchibald.github.io/svgomg/
But everytime you get less code then me😅😅😅

File: gravitymainnet/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+blockExplorers:  
+      - apiUrl: https://explorer.gravity.xyz/api
+        family: blockscout
+        name: gravityscan
+        url: https://explorer.gravity.xyz
+        chainId: 1625
```
[the remark this answers is not in the document]
Comment (JunayedMh) [reply gravitymainnet/metadata.yaml:6]: What i need to change?

**Pull request closed without merging**